applications/GeoMechanicsApplication/custom_elements/geo_steady_state_Pw_piping_element.h

```diff
@@ -32,12 +33,14 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoSteadyStatePwPipingElement : publ
 
     explicit GeoSteadyStatePwPipingElement(IndexType NewId = 0) : Element(NewId) {}
 
-    GeoSteadyStatePwPipingElement(IndexType NewId, GeometryType::Pointer pGeometry)
+    GeoSteadyStatePwPipingElement(IndexType NewId, const GeometryType::Pointer& pGeometry)
         : Element(NewId, pGeometry)
     {
     }
 
-    GeoSteadyStatePwPipingElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+    GeoSteadyStatePwPipingElement(IndexType                      NewId,
+                                  const GeometryType::Pointer&   pGeometry,
+                                  const PropertiesType::Pointer& pProperties)
```

Comment:
Same thing here:
```suggestion
                                  const GeometryType::Pointer&   rpGeometry,
                                  const PropertiesType::Pointer& rpProperties)
```
???

Reply:
took the suggested change.